src/views/dashboards/PasswordManager/BulkPasswordAccountModal.vue

```diff
@@ -0,0 +1,180 @@
+                  @change="parseFile")
+                button.btn.btn-primary(
+                  v-loading="globalLoading"
+                  :disabled="globalLoading"
+                  @click.prevent="triggerFile")
+                    | #[i.now-ui-icons.arrows-1_share-66] Upload CSV
+
+              div.mt-2.table-responsive
+                table.table.table-striped.table-bordered.m-0
+                  thead
+                    tr
+                      th Account Name
+                      th Username
+                      th Password
+                      th Additional Info
+                  tbody
+                    tr(v-if="!uploadedAccounts || uploadedAccounts.length === 0")
+                      td(colspan="100%")
+                        i No accounts uploaded yet...
+                    tr(v-for="account in uploadedAccounts")
+                      td {{ account.name }}
+                      td {{ account.username }}
+                      td {{ account.password }}
+                      td {{ account.info }}
+                
+              p.mt-2.text-center If all information looks correct above, click submit to store accounts on the blockchain!
+
+              div.d-flex.mt-2
+                div.ml-auto
+                  div.d-flex.justify-content-end
+                    button.btn.btn-primary.ml-auto(
+                      v-if="!needsToWriteDownPrivateKey"
+                      type="submit"
+                      v-loading="globalLoading"
+                      :disabled="globalLoading") Submit
+                    template(v-else)
+                      div
+                        div
+                          | We didn't find an encryption key to encrypt your account info with before
+                          | sending it to the blockchain. Therefore, we created one
+                        .my-3
+                          strong {{ encryptionKey }}
+                      button.btn.btn-lg.btn-danger.mx-auto(
+                        type="submit"
+                        v-loading="globalLoading"
+                        :disabled="globalLoading")
+                          | I wrote down my key and want to store this password on the blockchain!
+                  div.text-danger(v-if="!accountId")
+                    small
+                      div
+                        | You will spend #[strong {{ cost || `CAN'T CALCULATE` }} MTGY]
+                        | to store this account on the blockchain. 
+                      div 
+                        | It will not cost anything to
+                        | read or update it in the future.
+</template>
+
+<script>
+import { mapState } from "vuex";
+import FileUtils from "../../../factories/FileUtils";
+
+export default {
+  name: "BulkPasswordAccountModal",
+
+  data() {
+    return {
+      needsToWriteDownPrivateKey: false,
+      uploadedAccounts: [],
+    };
+  },
+
+  computed: mapState({
+    cost: (state) => state.passwordManager.cost,
```

Comment:
The cost is actually 50% cost * number of accounts.

Reply:
5790096